### PR TITLE
[DOC] Remove `..figure` directive in examples

### PR DESCRIPTION
**Pull request by skoudoro**
Sphinx gallery generate its own figure directive. So, currently our `..figure` directive are not processed and shows unfound files in our tutorials which is confusing. 

Furthermore, I added `.. rst-class:: centered small fst-italic fw-semibold` directive to simulate and not lose some captions.

### CODE REVIEW

**Comment by skoudoro** [comment]
Ok, this PR is ready to go.

is there any comments/suggestions ? I will go ahead and merge this tomorrow at noon if no comments. 

This cleaning was needed and more to come after the release.

File: doc/examples/bundlewarp_registration.py
Comment (arokem) [inline doc/examples/bundlewarp_registration.py:29]:
```suggestion
# Let's download and load two uncinate fasciculus bundles in the left
```

File: doc/examples/denoise_mppca.py
Comment (arokem) [inline doc/examples/denoise_mppca.py:17]:
```suggestion
or automatically based on the Marchenko-Pastur distribution [Veraa2016a]_.
```

File: doc/examples/denoise_mppca.py
Comment (arokem) [inline doc/examples/denoise_mppca.py:78]:
```suggestion
# The PCA denoising using the Marchenko-Pastur distribution can be performed by
```

File: doc/examples/denoise_mppca.py
Comment (arokem) [inline doc/examples/denoise_mppca.py:95]:
```suggestion
# To assess the performance of the Marchenko-Pastur PCA denoising algorithm,
```

File: doc/examples/denoise_mppca.py
Comment (arokem) [inline doc/examples/denoise_mppca.py:196]:
```suggestion
# Noise standard deviation estimation using the Marchenko-Pastur PCA algorithm
```

File: doc/examples/denoise_mppca.py
Comment (arokem) [inline doc/examples/denoise_mppca.py:218]:
```suggestion
# The above figure shows that the Marchenko-Pastur PCA algorithm computes a 3D
```

File: doc/examples/denoise_patch2self.py
Comment (arokem) [inline doc/examples/denoise_patch2self.py:154]:
```suggestion
# still get the same denoising performance. One can run Patch2Self
```

**Review by arokem — APPROVED** [review]
Looks good to me! I spotted a couple of typos along the way, but that doesn't mean there aren't any other ones :-)

**Comment by skoudoro** [comment]
thank you for the review @arokem!

For sure there is other ones, like old functions a simpler way to do stuff in many tutorials. We will continue to update that.during the year.

**Comment by skoudoro** [comment]
ps: building the doc in parallel, just waiting the build to finish before merging this PR.

**Pull request merged**
